cargo-process.el

```diff
@@ -275,10 +275,18 @@ Always set to nil if cargo-process--enable-rust-backtrace is nil"
     (let* ((metadata-text (shell-command-to-string
                            (concat (shell-quote-argument cargo-process--custom-path-to-bin)
                                    " metadata --format-version 1 --no-deps")))
-           (metadata-json (json-read-from-string metadata-text))
+           (metadata-json (cargo-json-read-from-string metadata-text))
            (workspace-root (cdr (assoc 'workspace_root metadata-json))))
       workspace-root)))
 
+(defun cargo-json-read-from-string (text)
```

Comment:
Can you place this `defun` above `cargo-process--workspace-root` ?

Reply:
done!